src/content/docs/en/guides/cms/payload.mdx

```diff
@@ -7,7 +7,215 @@ service: Payload CMS
 i18nReady: false
 ---
 
-[Payload](https://payloadcms.com/) is a headless CMS and application framework.
+[PayloadCMS](https://payloadcms.com/) is a headless open-source content management system that can be used to provide content for your Astro project.
+
+## Integrating with Astro
+
+## Prerequisites
```

Comment:
Are these supposed to be on the same H level?

Reply:
Nope, my bad! Fixed.